app/models.py

```diff
@@ -75,6 +77,19 @@ class Report(db.Model):
     date = db.Column(db.DateTime, nullable=False, default=datetime.now())
     comment = db.Column(db.String(1000), nullable=False)
 
+class EstateAgentRequest(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
+    estate_agent_id = db.Column(db.Integer, db.ForeignKey('estate_agent.id'), nullable=True)
+    name = db.Column(db.String(256), nullable=False)
+    email = db.Column(db.String(256), nullable=False)
+    phone = db.Column(db.String(256), nullable=False)
+    website = db.Column(db.String(256), nullable=False)
+    date = db.Column(db.DateTime, nullable=False, default=datetime.now())
+    handled = db.Column(db.Boolean, nullable=False, default=False)
+    user = db.relationship('User', back_populates='estate_agent_requests', lazy=True)
+    estate_agent = db.relationship('EstateAgent', back_populates='requests', lazy=True)
```

Comment:
Why would an estate agent request have a relationship with an existing estate agent? or am i just being stupid haha

Reply:
When the request is approved a new estate agent gets added and the original request is modified so it links to the new estate agent, not essential but could be useful to retrieve the linked estate agent from the request

Reply:
Ah nice one, makes sense